docs/user-documentation/metadata.md

```diff
@@ -1,50 +1,180 @@
 
+## Content Types
 
-## Fields
+In Drupal, _Content Types_ are entity subtypes that act as groupings of fields and their configurations, essentially a _metadata profile_, for content such as a digital object's descriptive record. The field configurations include how data is entered, how it is displayed, how many values can be stored, and how long the value can be. Some configurations, such as data entry and display, can be changed at any time. Others, such as how long a value can be, cannot be changed once content has been created using that content type.
 
-All normal field types are available to use for Islandora, including text, textarea,
- * do we need to warn against using HTML in textareas, in cases where it will be exported to Fedora?
+For example, the 'islandora_demo' module provides a _Repository Item_ content type that defines several fields including "Alternative Title" and "Date". The "Manage fields" page for Repository Item shows a list of the fields it includes as well as tabs for changing the input forms and display modes. The "Title" field does not appear in this list but it is still included with every content type. (See the "[Create / Update a Content Type](content_types.md)" section for more details on creating and configuring fields.)
 
-### Custom field: EDTF Date
+![Screenshot of the "Manage fields" page for the "Repository Item" content type from islandora_demo.](../assets/metadata_content_type_screenshot.png)
 
-### Custom field: Typed Relation
+> !!! tip: The included title field is limited to 255 characters; if your content has longer titles it is encouraged to create a separate long_title field to store the full title and reserve the default title field for a display title.
 
+> !!! tip "7.x Migration Note: What about my XML?"
+>
+> In 7.x, metadata were (usually) stored within XML datastreams such as MODS or DC. In CLAW we are breaking out
+individual metadata elements into fields instead of using an attached XML document. The Metadata Interest Group is developing a default mapping which will provide a basic, yet customizable, transform between MODS metadata and Drupal fields in Islandora Demo.
+>
+> It is still possible to attach an XML file to a CLAW object as a Media (see Datastreams), however there is no mechanism in CLAW for editing XML in a user-friendly way.
 
-## Mappings
+A specific instance of a content type is called a _node_, each of which has its own node identifier. In other words, a _node_ is the descriptive metadata for a particular digital object and the _content type_ is the node's metadata profile. Once a node is created, it cannot change its content type. To change a digital object's metadata profile (content type) a repository manager would need to create a new descriptive record (node) using the new metadata profile and then update the corresponding media records to point to the new descriptive record.
```

Comment:
Another streamlining suggestion.  You can drop the part about having an id here to read instead:

A specific instance of a content type is called a _node_. In other words, a _node_ is the descriptive metadata for a particular digital object and the _content type_ is the node's metadata profile.